src/components/formFields/createKeyValue/CreateKeyValue.tsx

```diff
@@ -133,25 +132,21 @@ const KeyValueComponent = ({
                 <TableCell>
                   <div className={styles.buttonsContainer}>
                     {copyValue && (
-                      <ToolTip tooltip="Copy value">
```

Comment:
This ToolTip now needs an `id` to know _where_ it should render. We do not currently have access to that `id` here. 

Two possible solutions:

1. Pass the `id` from the implementing application (in this case, I think, only the gateway-ui);
2. Create its own ToolTip.

Either way I thought it would be out of scope for this pull request.